README.md

```diff
@@ -98,142 +113,140 @@ oc login -u developer -p developer https://localhost:8443
+An official support for Apache Spark on OpenShift is provided by the [radanalytics.io](https://radanalytics.io/) project by means of
+the Oshinko application with a Web UI for deploying a Spark cluster. Other than using such a Web UI, a CLI tool is available as well which is used for this workshop.
 
-In the Keycloak UI, make sure you are in the 'default' realm. Then create a new user, and a set of credentials for that user. Make sure the user is
-enabled, and that the credentials are not marked as temporary.
+Go to [Oshinko CLI downloads](https://github.com/radanalyticsio/oshinko-cli/releases) and download
+the latest release (0.3.1 as of time of writing). Unpack the release:
 
-For this workshop we could use following users for example :
+```
+tar xvf oshinko_v0.3.1_linux_amd64.tar.gz
+```
 
-* _console_ : as a "real" user who uses the console for creating addresses
-* _deviceX_ : as deviceX (i.e. device1, device2, ...) user who can only send/recv from a particular address
-* _sparkdriver_ : as Spark driver application user 
-* _thermostat_ : as thermostat application user
+Then use the following command in order to deploy a Spark cluster made by one master node and one slave node.
 
-It's clear that this workshop involves a "real" user who is in charge to create and configure the addresses and some other users which are related to applications running in the cluster and devices running in the field.
+```
+export SPARK_NAME=<something>
+
+oc new-project spark
+./oshinko_linux_amd64/oshinko create $SPARK_NAME --masters=1 --workers=1
+```
+
+## IoT Application
+
+Now that the cluster has been set up, its time to work on the example application. First, we will
+provision messaging infrastructure to use with the application.
+
+### Provisioning messaging
+
+Go to the OpenShift Console.
 
-It is now time to create some authorization rules to ensure the different users can only access the
-addresses assigned. This is done by creating groups with names on the form `send_myqueue` and have a
-user join that group to gain the permission (to send messages to address `myqueue`). We can for
-example create the following groups that will match the addresses that we will create later:
+In the OpenShift Service Catalog overview, select either of "EnMasse (standard)" or "EnMasse (brokered)".
 
-* manage
-* send\_temperature
-* recv\_temperature
-* send\_max
-* recv\_max
-* send\_control
-* recv\_control
+![Catalog](images/catalog.png)
 
-To enable authorization, we need to have the users join the appropriate groups. For this workshop we
-can use the following mapping:
+Select among the available plans. Select the "Create Project" in the drop-down box.
 
-* console - manage
-* deviceX - send\_temperature, recv\_control/deviceX
-* sparkdriver - recv\_temperature, send\_max
-* thermostat - recv\_max, send\_control*
+Use the same value for the "name" field. The address space will be provisioned and may take a few
```

Comment:
Did you mean "Use the same value for the "name" field and the "Project name" field" ? (as I see from the image)

Reply:
yes